tests/tests.yaml

```diff
@@ -6,6 +6,6 @@ target_deploy_status:
     workload-status: active
     workload-status-message: icmp ok, local hostname ok, dns ok
 configure:
-- zaza.openstack.charm_tests.noop.setup.basic_setup
+- zaza.charm_tests.noop.setup.basic_setup
 tests:
-- zaza.openstack.charm_tests.noop.tests.NoopTest
+- zaza.charm_tests.noop.tests.NoopTest
```

Comment:
This has been wrong for a year and there is no tox target pointing to this `tests/` folder, although the readme in this folder says:

> This directory contains functional test definition for functional test of Zaza
> itself.

Is this `tests/` folder really still valid?

Reply:
This is absolutely 100% a correct import path. This is coming from zaza itself, rather than from zaza.openstack

Reply:
@ChrisMacNaughton @fnordahl As discussed offline I got rid of the entire `tests/` folder for now. 

For the record what we noticed is:
1. `tox.ini` doesn't have a `func` target so probably these tests haven't been run for a long time.
2. These tests can't be run at the moment because of conflicting version ranges in python deps pulled via `requirements.txt` and `setup.py`, e.g.

```
error: PyYAML 3.13 is installed but pyyaml<=6.0,>=5.1.2 is required by {'juju'}
error: python-keystoneclient 3.21.0 is installed but python-keystoneclient>=3.22.0 is required by {'python-openstackclient'}
```

Reply:
I'm not sure why the `tests/` directory is being removed as part of the PR?  It may be that it *should* be removed, but it shouldn't be done as part of this PR unless it is directly relevant to the functionality that is being tested (neutron gateway managed bridges).

The reason is, should we wish to re-instate the `tests` directory it would be more awkward as it is mixed in with this functionality.

Reply:
Thanks, makes sense, moved [here](https://github.com/openstack-charmers/zaza-openstack-tests/pull/219)